pkgs/applications/science/astronomy/gildas/default.nix

```diff
@@ -0,0 +1,68 @@
+{ stdenv, fetchurl, gtk2-x11 , pkgconfig , python27 , gfortran , lesstif
+, cfitsio , getopt , perl , groff , which , openblas
+}:
+
+let
+  python27Env = python27.withPackages(ps: with ps; [ numpy ]);
+in
+
+stdenv.mkDerivation rec {
+  srcVersion = "jun18a";
+  version = "20180601_a";
+  name = "gildas-${version}";
+
+  src = fetchurl {
+    url = "http://www.iram.fr/~gildas/dist/gildas-src-${srcVersion}.tar.gz";
+    sha256 = "0k4x0g69fphb1759cwcw6bbs8imwmq0qwj6zqixxk60skk4n4jvb";
+  };
+
+  enableParallelBuilding = true;
+
+  nativeBuildInputs = [ pkgconfig groff perl getopt gfortran which ];
+
+  buildInputs = [ gtk2-x11 lesstif cfitsio python27Env openblas ];
+
+  patches = [ ./wrapper.patch ./return-error-code.patch ./openblas.patch ./clang.patch ];
+
+  configurePhase=''
+    substituteInPlace admin/wrapper.sh --replace '%%OUT%%' $out
+    substituteInPlace admin/wrapper.sh --replace '%%PYTHONHOME%%' ${python27Env}
+    source admin/gildas-env.sh -c gfortran -o openmp -s ${openblas}/lib
+    echo "gag_doc:        $out/share/doc/" >> kernel/etc/gag.dico.lcl
+  '';
+
+  buildPhase=''
+    make depend
+    make install
+  '';
+
+  installPhase=''
```

Comment:
Can this be moved to the `postInstallPhase`? Then it looks like the standard `installPhase` can call `make install`, which is currently in the `buildPhase`.

Reply:
Done in 8f8a251